upload.mk

```
@@ -32,7 +32,7 @@ endif
 	  file="$$(git ls-files "$${1%-[0-9][0-9]}.*")"; \
 	  for last in $$(git log --follow --name-only --format=format: -- "$${file%-[0-9][0-9]}" | \
 		sed -e '/^$$/d' | grep -v draft-todo-yourname-protocol | cut -f 2 | uniq | tail +2); do \
-	    if [ -n "$$(git tag -l "$${last%.*}-*")" ]; then \
+	    if [ -n "$$(git tag -l | grep -E "^$${last%.*}-[0-9]{2}$$")" ]; then \
```

```suggestion
	    if [ -n "$$(git tag -l "$${last%.*}-[0-9][0-9]")" ]; then \
```

You might have missed the simpler fix...

---

I thought about it, but that matches the tag `aaaaaaa-draft-stuff-foobar-03`. Not sure if that matters

---

I don't think it does.  If `git tag -l a` doesn't match "ab", then this won't match `aaaaa-draft-whatever-09`.

---

Oh. Duh